website/docs/Use-Cases/agent_chat.md

```diff
@@ -285,6 +285,35 @@ By adopting the conversation-driven control with both programming language and n
 - LLM-based function call. In this approach, LLM decides whether or not to call a particular function depending on the conversation status in each inference call.
   By messaging additional agents in the called functions, the LLM can drive dynamic multi-agent conversation. A working system showcasing this type of dynamic conversation can be found in the [multi-user math problem solving scenario](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_two_users.ipynb), where a student assistant would automatically resort to an expert using function calls.
 
+## LLM Caching
```

Comment:
level 2 heading here and level 3 heading next (Diverse Applications...) are confusing. What about using level 3 heading?
Our current documentation about caching is under enhanced_inference.md. The doc in that file needs to be modified accordingly.

Reply:
Fixed heading.